app/controllers/posts_controller.rb

```diff
@@ -1,7 +1,7 @@
 class PostsController < ApplicationController
   def index
-    @user = User.find_by(id: params[:user_id])
-    @posts = @user.posts
+    @user = User.includes(:posts).find_by(id: params[:user_id])
+    @posts = @user.posts.includes(:comments, :lkes)
```

Comment:
- [ ] Good job for implementing the previous reviewer's suggestion 👍 However, please make sure to spell the `likes` correctly so that the app behaves as expected. Currently I'm getting the following error ⚠👇
 
 ![image](https://github.com/momo-87/blog-app/assets/106119105/af98c64b-d5fe-4e5b-925f-005bc32ef40a)


- [ ] Similarly, [as suggested](https://github.com/momo-87/blog-app/pull/8#discussion_r1308471403), please make sure to attach screenshots of the terminal logs before and after fixing the _N+1 queries_ in the pull request summary 💪